app/routes.py

```diff
@@ -1,5 +1,5 @@
 from flask import request, jsonify, session, render_template
-from app import app
+# from app import app
```

Comment:
was getting a circular import error. Not 100% sure this will solve the problem or create more